src/helpers/metrics.ts

```diff
@@ -41,6 +45,39 @@ export default function initMetrics(app: Express) {
   });
 
   app.use(instrumentRateLimitedRequests);
+
+  app.use((req: Request, res: Response, next) => {
+    if (req.originalUrl.startsWith('/graphql')) {
+      const end = graphqlRequest.startTimer();
```

Comment:
We are not ending this if there is no query or operationName 
```javascript
if (query && operationName) {
```

Reply:
We are tracking only valid queries

Reply:
oh i thought it would stay in memory for ever 🤔